conf/ocsci/multus_public_network_interface_412_vsphere.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# Use this config for enabling multus deployment on vsphere for OCP 4.12
+ENV_DATA:
+  is_multus_enabled: true
+  multus_public_network_interface: 'br-ex'
```

Comment:
Is this something we need to create the on the vSphere level manually in the specific DC before the deployment? Or this is automatically created?

Reply:
it will be created automatically by openshift. we don't need to create anything